tests/test_server.py

```diff
@@ -249,6 +252,33 @@ def test_evaluate(app):
                                                              "policy"}
 
 
+def test_stack_training(app,
+                        default_domain_path,
+                        default_stories_file,
+                        default_stack_config,
+                        default_nlu_data
+                        ):
+    payload = dict(
+        domain=open(default_domain_path).read(),
```

Comment:
let's make sure these file handles are closed - it is a bit more verbose, but avoids warnings about unclosed handles:
```
domain_file = open(default_domain_path)
payload = ...
domain_file.close()
```